Analysis/Tasks/PWGHF/taskJpsi.cxx

```diff
@@ -36,53 +39,55 @@ void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 struct TaskJpsi {
   HistogramRegistry registry{
     "registry",
-    {{"hmass", "2-prong candidates;inv. mass (#e+ e-) (GeV/#it{c}^{2});entries", {HistType::kTH1F, {{500, 0., 5.}}}},
-     {"hptcand", "2-prong candidates;candidate #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
+    {{"hptcand", "2-prong candidates;candidate #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
      {"hptprong0", "2-prong candidates;prong 0 #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
-     {"hptprong1", "2-prong candidates;prong 1 #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}},
-     {"hdeclength", "2-prong candidates;decay length (cm);entries", {HistType::kTH1F, {{200, 0., 2.}}}},
-     {"hdeclengthxy", "2-prong candidates;decay length xy (cm);entries", {HistType::kTH1F, {{200, 0., 2.}}}},
-     {"hd0Prong0", "2-prong candidates;prong 0 DCAxy to prim. vertex (cm);entries", {HistType::kTH1F, {{100, -1., 1.}}}},
-     {"hd0Prong1", "2-prong candidates;prong 1 DCAxy to prim. vertex (cm);entries", {HistType::kTH1F, {{100, -1., 1.}}}},
-     {"hd0d0", "2-prong candidates;product of DCAxy to prim. vertex (cm^{2});entries", {HistType::kTH1F, {{500, -1., 1.}}}},
-     {"hCPA", "2-prong candidates;cosine of pointing angle;entries", {HistType::kTH1F, {{110, -1.1, 1.1}}}},
-     {"hEta", "2-prong candidates;candidate #it{#eta};entries", {HistType::kTH1F, {{100, -2., 2.}}}},
-     {"hImpParErr", "2-prong candidates;impact parameter error (cm);entries", {HistType::kTH1F, {{100, -1., 1.}}}},
-     {"hDecLenErr", "2-prong candidates;decay length error (cm);entries", {HistType::kTH1F, {{100, 0., 1.}}}},
-     {"hDecLenXYErr", "2-prong candidates;decay length xy error (cm);entries", {HistType::kTH1F, {{100, 0., 1.}}}}}};
+     {"hptprong1", "2-prong candidates;prong 1 #it{p}_{T} (GeV/#it{c});entries", {HistType::kTH1F, {{100, 0., 10.}}}}}};
 
   Configurable<int> d_selectionFlagJpsi{"d_selectionFlagJpsi", 1, "Selection Flag for Jpsi"};
   Configurable<double> cutYCandMax{"cutYCandMax", -1., "max. cand. rapidity"};
+  Configurable<std::vector<double>> bins{"pTBins", std::vector<double>{hf_cuts_jpsi_toee::pTBins_v}, "pT bin limits"};
+  void init(o2::framework::InitContext&)
+  {
+    registry.add("hmass", "2-prong candidates;inv. mass (#e+ e-) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{500, 0., 5.}, {(std::vector<double>)bins, "#it{p}_{T} (GeV/#it{c})"}}});
```

Comment:
You can also cast it once with `auto vbins = (std::vector<double>)bins;` and use `vbins` in further statements. In the future we will get a special configurable axis spec type so it would be easier to change the type only in one place later.

Reply:
> You can also cast it once with `auto vbins = (std::vector<double>)bins;` and use `vbins` in further statements. In the future we will get a special configurable axis spec type so it would be easier to change the type only in one place later.

Ok, So here I just keep it at the moment? Or change it?

Reply:
No need to change now, this is just for you information. I do not have any other comments, so if you have already tested that it compiles and runs (and reproduces the results) you can ask @ginnocen to mark this as non-draft.

Reply:
> No need to change now, this is just for you information. I do not have any other comments, so if you have already tested that it compiles and runs (and reproduces the results) you can ask @ginnocen to mark this as non-draft.

Ok, thanks a lot! I have already tested it in the local, and It can be compiled and reproduces the results. @ginnocen

Reply:
Fine with me for now. But in the next iteration, I would suggest to adjust it to match the simpler version with `vbins` in Anton's PR.